### DISCUSSION TITLE
Sparklines?

### DISCUSSION THREAD

**Discussion by kevinchiu** [achannarasappa/ticker#25] [Ideas]
Would it be possible to add sparklines to this?

**Comment by achannarasappa** [comment]
Interesting idea! From a visual perspective, it would be possible with the [Unicode Braille characters](https://en.wikipedia.org/wiki/Braille_Patterns) however from a data perspective, the app would need to either store historical data or do an initial pull of historical data and append to that. There's also some responsive design considerations that would need to be accounted in supporting multiple terminal widths.

It's an idea worth thinking through but the particulars would determine the technical and design feasibility.
